src/gui/codeeditors/qgscodeeditorpython.h

```diff
@@ -84,10 +84,24 @@ class GUI_EXPORT QgsCodeEditorPython : public QgsCodeEditor
      */
     void searchSelectedTextInPyQGISDocs();
 
+    /**
+     * Toggle comment for the selected text.
+     *
+     * \since QGIS 3.30
+     */
+    void toggleComment();
+
   protected:
 
     void initializeLexer() override;
 
+    /**
+     * Override QgsCodeEdito::keyPressEvent to provide custom shortcuts
+     *
+     * \since QGIS 3.30
+     */
```

Comment:
```suggestion
```

No doxygen is necessary for overridden methods

Reply:
Duly noted